tests/test_webhook.py

```diff
@@ -1,42 +1,52 @@
-# Unit tests related to 'Webhook' (https://www.easypost.com/webhooks-guide).
-
 import pytest
 
 import easypost
 
 
 @pytest.mark.vcr()
-def test_webhooks(per_run_unique):
-    url = "example.com/{0}".format(per_run_unique)
-    expected_url = "http://example.com/{0}".format(per_run_unique)
-
-    # Create a webhook
-    webhook = easypost.Webhook.create(url=url)
-    assert webhook.id is not None
-    assert webhook.mode == "test"
-    assert webhook.url == expected_url
-    assert webhook.disabled_at is None
+def test_webhook_create():
+    webhook = easypost.Webhook.create(url="http://example.com")
```

Comment:
Nit: You could make the webhook url a fixture like the others so you can reuse it and if ever we needed to switch it it's easier.